internal-services/catalog/iib-add-fbc-fragment-to-index-image-task.yaml

```diff
@@ -232,8 +232,8 @@ spec:
             url=\$(jq -r ".logs.url" <<< \${build_info})
             echo IIB log url is: "\${url}" > \$(results.iibLog.path)
             echo Printing last 2,000 characters from the log >> \$(results.iibLog.path)
```

Comment:
this seems wrong

Reply:
Thanks, fixed. Missed it because of the `,`